dts/bindings/display/led-strip-matrix.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Generic LED strip matrix (LED strip arranged in a grid pattern)
+
+compatible: "led-strip-matrix"
+
+include: display-controller.yaml
+
+properties:
+  led-strip:
+    type: phandle
+    required: true
+    description: |
+      Specify the LED strip that is the substance of the matrix.
+
+  circulative:
+    type: boolean
+    description: |
+      Specify this when the LED layout is not wrapped
+      but follows the coordinate arrangement.
```

Comment:
the wording is not clear here unless one looks at the sample that follows. It should be reworded so that it is clearer what happens when true vs. false, where here one has to go through a double negation etc.

Reply:
I tried rewriting it. Could you please take a look?